include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -188,6 +188,50 @@ class CommonOperatorLoader : public ProtobufLoader {
     addNodeAsOutput(op, node);
   }
 
+  void loadBatchMatMul(const OpType &op, ArgumentDictionaryTy &dict) {
+    const std::string &opName = loadOperatorName(op);
+    auto LHS = getNodeValueOrCreateConstantByName(op.input(0));
+    auto RHS = getNodeValueOrCreateConstantByName(op.input(1));
+
+    bool transLHS = dict.count("trans_a") && (loadInt(dict["trans_a"]) == 1);
+    (void)transLHS;
+    assert(!transLHS && "Don't support transpose lhs for now.");
```

Comment:
can you reflect this properly, e.g., returning false

Reply:
I can return `false` but how am I going to clearly issue the error message?

Reply:
This is a different story. I suggest not to worry about it in this PR.